packages/main/src/Dialog.js

```diff
@@ -77,75 +84,75 @@ class Dialog extends Popup {
 	}
 
 	static get styles() {
-		return [Popup.styles, dialogCss];
+		return [dialogCSS, popupCSS];
+	}
+
+	constructor() {
+		super();
+
+		// mark dialog for registry as it is always modal
+		this.modal = true;
```

Comment:
I put the comment here, but it's for Popup, dialog and popover. You have a modal private property on Popup.js level and then on Popover.js level you have modal metadata var, which will create a get modal() getter. This may become confusing. Please create a: `get isModal()` getter on Popup.js/Popover.js/Dialog.js. For Dialog hard-code it to return true, for popover let it return the value of the modal metadata property. Use this isModal getter everywhere in the low-level code and let there be no confusion with the popover property.